src/Ticket.php

```diff
@@ -5486,6 +5486,11 @@ public static function showListForItem(CommonDBTM $item, $withtemplate = 0)
                                 'glpi_tickets_users.tickets_id'  => new \QueryExpression('glpi_tickets.id'),
                                 'glpi_tickets_users.users_id'    => Session::getLoginUserID()
                             ]
+                        ], [
+                            'AND' => [
+                                'glpi_ticketvalidations.tickets_id'        => new \QueryExpression('glpi_tickets.id'),
```

Comment:
This line is also redundant with the ON clause of a LEFT JOIN. I keep it for consistency with the other condition.

However, I think it could be removed (same for the other one)